benchmarks/user/README.md

```diff
@@ -0,0 +1,233 @@
+# Benchmarking a Devito Operator
+
+## Running benchmarks
+
+`benchmark.py` implements a minimalist framework to evaluate the performance of
+a Devito Operator while varying
+
+* the problem size (e.g., shape of the computational grid)
```

Comment:
Trivial :
Maybe size and shape/ size of the shape...
* the problem size (e.g., shape size of the computational grid)
I know it is defined as `shape` but sounds like weird IMHO....

Reply:
mmhh, to be fair, I think it's quite clear the way it is